src/main/scala/scala/tools/refactoring/analysis/IsNotInImports.scala

```diff
@@ -0,0 +1,53 @@
+package scala.tools.refactoring
+package analysis
+
+trait IsNotInImports { _: CompilationUnitDependencies with common.EnrichedTrees =>
```

Comment:
As already pointed out, to me `IsNotInImports` is the name of a method, and not something that should have it's own trait. I'm generally not very fond of using stackable traits as the primary way to reuse code, as they often make code quite hard to reason about. They are also causing lot's of recompiles during editing, which can be quite annoying. Of course, we have to bear them because the compiler API forces their cakes on us, but we should bear them out of necessity, not out of enthusiasm. To me, the good old advice "Prefer composition over inheritance" also applies to Scala.